app/src/main/java/com/greenfox/kalendaryo/models/KalUser.java

```diff
@@ -0,0 +1,27 @@
+package com.greenfox.kalendaryo.models;
+
+/**
+ * Created by Ezzo on 2017. 12. 15..
+ */
+
+public class KalUser {
+    public long id;
+    public String accessToken;
```

Comment:
Based on the docuentation these fields should be:
    private String userId;
    private String clientToken;
    private String authId;
    private String accessToken;
Please rename!